From 85710dd8f749a447301e1522e6022b62e29d6f5d Mon Sep 17 00:00:00 2001
From: Chase Maupin <Chase.Maupin@ti.com>
Date: Wed, 9 Feb 2011 14:20:51 -0600
Subject: [PATCH] cups desktop: fix cups desktop file

* Fix the cups.desktop file to address issues reported by the
  desktop-file-validate program.  This resolves QA error
  issues generated by the insane class:
    ERROR: QA Issue with cups: cups.desktop: warning: value
        "Application;System;X-Red-Hat-Base;" for key "Categories" in group "
        Desktop Entry" contains a deprecated value "Application"
    ERROR: QA Issue with cups: cups.desktop: warning: key "Encoding" in group
        "Desktop Entry" is deprecated
    ERROR: QA Issue with cups: cups.desktop: warning: value "" for key
        "Path" in group "Desktop Entry" does not look like an absolute path
    ERROR: QA Issue with cups: cups.desktop: warning: key "TerminalOptions"
        in group "Desktop Entry" is deprecated
* This patch was obtained from http://bugs.gentoo.org/show_bug.cgi?id=236706
  and thank to Pacho Ramos <pacho@gentoo.org> for the fix

Signed-off-by: Chase Maupin <Chase.Maupin@ti.com>
---
 desktop/cups.desktop |    5 +----
 1 files changed, 1 insertions(+), 4 deletions(-)

diff --git a/desktop/cups.desktop b/desktop/cups.desktop
index dabfba3..28b98d9 100644
--- a/desktop/cups.desktop
+++ b/desktop/cups.desktop
@@ -1,14 +1,11 @@
 [Desktop Entry]
-Categories=Application;System;X-Red-Hat-Base;
-Encoding=UTF-8
+Categories=System;X-Red-Hat-Base;
 Exec=htmlview http://localhost:631/
 GenericName=
 Icon=cups
 MimeType=
-Path=
 StartupNotify=false
 Terminal=false
-TerminalOptions=
 Type=Application
 X-DCOP-ServiceType=
 X-KDE-SubstituteUID=false
-- 
1.7.0.4

